<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gmat</class>
 <widget class="QDialog" name="gmat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>531</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*-----QWidget-----*/

QWidget

{

	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(27, 39, 50, 255),stop:1 rgba(47, 53, 74, 255));

	color: #000000;



}





/*-----QLabel-----*/

QLabel

{

	background-color: transparent;

	color: #c2c7d5;

	font-size: 13px;



}





/*-----QPushButton-----*/

QPushButton

{

	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 172, 149, 255),stop:0.995192 rgba(54, 197, 177, 255));

	color: #fff;

	font-size: 11px;

	font-weight: bold;

	border: none;

	border-radius: 3px;

	padding: 5px;



}





QPushButton::pressed

{

	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 207, 179, 255),stop:1 rgba(70, 255, 230, 255));



}





/*-----QCheckBox-----*/

QCheckBox

{

	background-color: transparent;

	color: #fff;

	font-size: 10px;

	font-weight: bold;

	border: none;

	border-radius: 5px;



}





/*-----QCheckBox-----*/

QCheckBox::indicator

{

    color: #b1b1b1;

    background-color: #323232;

    border: 1px solid darkgray;

    width: 12px;

    height: 12px;



}





QCheckBox::indicator:checked

{

    image:url(&quot;./ressources/check.png&quot;);

	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 172, 149, 255),stop:0.995192 rgba(54, 197, 177, 255));;

    border: 1px solid #607cff;



}





QCheckBox::indicator:unchecked:hover

{

    border: 1px solid #08b099;



}





QCheckBox::disabled

{

	color: #656565;



}





QCheckBox::indicator:disabled

{

	background-color: #656565;

	color: #656565;

    border: 1px solid #656565;



}





/*-----QLineEdit-----*/

QLineEdit

{

	background-color: #c2c7d5;

	color: #000;

	font-weight: bold;

	border: none;

	border-radius: 2px;

	padding: 3px;



}

QTextEdit { color: white }





/*-----QListView-----*/

QListView

{

	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(50, 61, 80, 255),stop:1 rgba(44, 49, 69, 255));

	color: #fff;

	font-size: 12px;

	font-weight: bold;

	border: 1px solid #191919;

	show-decoration-selected: 0;



}





QListView::item

{

	color: #31cecb;

	background-color: #454e5e;

	border: none;

	padding: 5px;

	border-radius: 0px;

	padding-left : 10px;

	height: 42px;



}



QListView::item:selected

{

	color: #31cecb;

	background-color: #454e5e;



}





QListView::item:!selected

{

	color:white;

	background-color: transparent;

	border: none;

	padding-left : 10px;



}





QListView::item:!selected:hover

{

	color: #bbbcba;

	background-color: #454e5e;

	border: none;

	padding-left : 10px;



}





/*-----QTreeView-----*/

QTreeView 

{

	background-color: #232939;

	show-decoration-selected: 0;

	color: #c2c8d7;



}





QTreeView::item 

{

	border-top-color: transparent;

	border-bottom-color: transparent;



}





QTreeView::item:hover 

{

	background-color: #606060;

	color: #fff;



}





QTreeView::item:selected 

{

	background-color: #0ab19a;

	color: #fff;



}





QTreeView::item:selected:active

{

	background-color: #0ab19a;

	color: #fff;



}





QTreeView::branch:has-children:!has-siblings:closed,

QTreeView::branch:closed:has-children:has-siblings 

{

	image: url(://tree-closed.png);



}



QTreeView::branch:open:has-children:!has-siblings,

QTreeView::branch:open:has-children:has-siblings  

{

	image: url(://tree-open.png);



}





/*-----QTableView &amp; QTableWidget-----*/

QTableView

{

    background-color: #232939;

	border: 1px solid gray;

    color: #f0f0f0;

    gridline-color: #232939;

    outline : 0;



}





QTableView::disabled

{

    background-color: #242526;

    border: 1px solid #32414B;

    color: #656565;

    gridline-color: #656565;

    outline : 0;



}





QTableView::item:hover 

{

    background-color: #606060;

    color: #f0f0f0;



}





QTableView::item:selected 

{

	background-color: #0ab19a;

    color: #F0F0F0;



}





QTableView::item:selected:disabled

{

    background-color: #1a1b1c;

    border: 2px solid #525251;

    color: #656565;



}





QTableCornerButton::section

{

	background-color: #343a49;

    color: #fff;



}





QHeaderView::section

{

	color: #fff;

	border-top: 0px;

	border-bottom: 1px solid gray;

	border-right: 1px solid gray;

	background-color: #343a49;

    margin-top:1px;

	margin-bottom:1px;

	padding: 5px;



}





QHeaderView::section:disabled

{

    background-color: #525251;

    color: #656565;



}





QHeaderView::section:checked

{

    color: #fff;

    background-color: #0ab19a;



}





QHeaderView::section:checked:disabled

{

    color: #656565;

    background-color: #525251;



}





QHeaderView::section::vertical::first,

QHeaderView::section::vertical::only-one

{

    border-top: 1px solid #353635;



}





QHeaderView::section::vertical

{

    border-top: 1px solid #353635;



}





QHeaderView::section::horizontal::first,

QHeaderView::section::horizontal::only-one

{

    border-left: 1px solid #353635;



}





QHeaderView::section::horizontal

{

    border-left: 1px solid #353635;



}





/*-----QScrollBar-----*/

QScrollBar:horizontal 

{

    background-color: transparent;

    height: 8px;

    margin: 0px;

    padding: 0px;



}





QScrollBar::handle:horizontal 

{

    border: none;

	min-width: 100px;

    background-color: #56576c;



}





QScrollBar::add-line:horizontal, 

QScrollBar::sub-line:horizontal,

QScrollBar::add-page:horizontal, 

QScrollBar::sub-page:horizontal 

{

    width: 0px;

    background-color: transparent;



}





QScrollBar:vertical 

{

    background-color: transparent;

    width: 8px;

    margin: 0;



}





QScrollBar::handle:vertical 

{

    border: none;

	min-height: 100px;

    background-color: #56576c;



}





QScrollBar::add-line:vertical, 

QScrollBar::sub-line:vertical,

QScrollBar::add-page:vertical, 

QScrollBar::sub-page:vertical 

{

    height: 0px;

    background-color: transparent;



}

QGroupBox  {

    border: 1px solid gray;

    border-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 172, 149, 255),stop:0.995192 rgba(54, 197, 177, 255));;

    margin-top: 27px;

    font-size: 14px;

    border-radius: 15px;

}



QGroupBox::title  {

    subcontrol-origin: margin;

    subcontrol-position: top center;

    padding: 5px 8000px 5px 8000px;

    background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 172, 149, 255),stop:0.995192 rgba(54, 197, 177, 255));;

    color: rgb(255, 255, 255);

}

QLineEdit {

    border: 2px solid gray;

    border-radius: 10px;

    padding: 0 8px;

    background: white;

    selection-background-color: darkgray;

}

QComboBox {

    border: 2px solid grey;

    border-radius: 10px;

    padding: 0 8px;

    min-width: 6em;

}



QComboBox:editable {

    background: white;

}



QComboBox:!editable, QComboBox::drop-down:editable {

     background: white;

}



/* QComboBox gets the &quot;on&quot; state when the popup is open */

QComboBox:!editable:on, QComboBox::drop-down:editable:on {

    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,

                                stop: 0 #D3D3D3, stop: 0.4 #D8D8D8,

                                stop: 0.5 #DDDDDD, stop: 1.0 #E1E1E1);

}



QComboBox:on { /* shift the text when the popup opens */

    padding-top: 3px;

    padding-left: 4px;

}



QComboBox::drop-down {

    subcontrol-origin: padding;

    subcontrol-position: top right;

    width: 15px;



    border-left-width: 1px;

    border-left-color: darkgray;

    border-left-style: solid; /* just a single line */

    border-top-right-radius: 10px; /* same radius as the QComboBox */

    border-bottom-right-radius: 10px;

}



QComboBox::down-arrow {

    image: url(/usr/share/icons/crystalsvg/16x16/actions/1downarrow.png);

}



QComboBox::down-arrow:on { /* shift the arrow when popup is open */

    top: 1px;

    left: 1px;

}

QComboBox QAbstractItemView {

    border: 2px solid gray;

    selection-background-color: lightgray;

}

QTabWidget::pane { /* The tab widget frame */

    border-top: 2px solid #C2C7CB;

}



QTabWidget::tab-bar {

    left: 5px; /* move to the right by 5px */

}



/* Style the tab using the tab sub-control. Note that

    it reads QTabBar _not_ QTabWidget */

QTabBar::tab {

    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,

                                stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,

                                stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);

    border: 2px solid qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 172, 149, 255),stop:0.995192 rgba(54, 197, 177, 255));;

    border-bottom-color: #C2C7CB; /* same as the pane color */

    border-top-left-radius: 4px;

    border-top-right-radius: 4px;

    min-width: 8ex;

    padding: 2px;

}



QTabBar::tab:selected, QTabBar::tab:hover {

    background: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 172, 149, 255),stop:0.995192 rgba(54, 197, 177, 255));

}



QTabBar::tab:selected {

    border-color:qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 172, 149, 255),stop:0.995192 rgba(54, 197, 177, 255));

    border-bottom-color: #C2C7CB; /* same as pane color */

}



QTabBar::tab:!selected {

    margin-top: 2px; /* make non-selected tabs look smaller */

}



/* make use of negative margins for overlapping tabs */

QTabBar::tab:selected {

    /* expand/overlap to the left and right by 4px */

    margin-left: -4px;

    margin-right: -4px;

}



QTabBar::tab:first:selected {

    margin-left: 0; /* the first selected tab has nothing to overlap with on the left */

}



QTabBar::tab:last:selected {

    margin-right: 0; /* the last selected tab has nothing to overlap with on the right */

}



QTabBar::tab:only-one {

    margin: 0; /* if there is only one tab, we don't want overlapping margins */

}

QRadioButton{



	color:white;

	font:solid;

}

QDateEdit

{

    background-color: white;

	border: 2px solid grey;

    border-radius: 10px;

    padding: 0 8px;

    spacing: 5px; 

    min-width: 6em;

}





QDateEdit::drop-down {

    image: url(:/new/myapp/cbarrowdn.png);

    width:50px;

    height:15px;

    subcontrol-position: right top;

    subcontrol-origin:margin;

    background-color: white;

   	border: 2px solid grey;

    border-radius: 10px;

    padding: 0 8px;

    spacing: 5px; 

    min-width: 6em;

   spacing: 5px; 

}

</string>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Gemployee</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>GProduits</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>GArticles</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="Gclient">
    <attribute name="title">
     <string>GMateriels</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>11</y>
       <width>371</width>
       <height>438</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>search</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_rech_mat"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tab_mat">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QPushButton" name="pdfmat">
            <property name="text">
             <string>print</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_9">
            <property name="text">
             <string>excel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QPushButton" name="pushButton_5">
            <property name="text">
             <string>show all</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_tri_mat">
            <property name="text">
             <string>tri</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>120</y>
       <width>421</width>
       <height>271</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>id_mat</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="id_mat"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>name_mat</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="nom_mat"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QComboBox" name="etat_mat">
            <property name="editable">
             <bool>true</bool>
            </property>
            <property name="currentText">
             <string>etat</string>
            </property>
            <item>
             <property name="text">
              <string>etat</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>a reparer</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>en utilisation </string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>libre</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="gamme_mat">
            <property name="editable">
             <bool>true</bool>
            </property>
            <property name="currentText">
             <string>gamme</string>
            </property>
            <item>
             <property name="text">
              <string>gamme</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>bas </string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>milieu </string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>haut</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="type_mat">
            <property name="editable">
             <bool>true</bool>
            </property>
            <property name="currentText">
             <string>type</string>
            </property>
            <item>
             <property name="text">
              <string>type</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Table à repasser</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Cabine de détachage</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Mannequin de repassage</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Sécheuse-repasseuse</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Séchoir électrique</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Machine à laver</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>fer a repasser</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="ajout_mat">
            <property name="text">
             <string>add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>search</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="modif_mat">
            <property name="text">
             <string>modify</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="supp_mat">
            <property name="text">
             <string>supprimer</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>10</y>
       <width>151</width>
       <height>56</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_11">
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>Quit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>10</y>
       <width>182</width>
       <height>24</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>id_mat</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_modif_mat"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>470</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>porte</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_info_mat">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>400</y>
       <width>401</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>GClients</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>GFournisseurs</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>mail</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>server port</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>140</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>username</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>170</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>password</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>200</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>to</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>230</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>subject</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>smtp-server</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="server">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="port">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="uname">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>140</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="paswd">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>170</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="rcpt">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>200</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="subject">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>230</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>380</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>send</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="msg">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>286</y>
       <width>391</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
